requirements.txt

```diff
@@ -10,7 +10,7 @@ ansi2html
 dash_html_components==1.1.4
 pandas==1.2.4
 dash_core_components==1.17.1
-plotly==5.2.1
+plotly>=5.2.1
```

Comment:
I changed this because a more recent version is listed in the `Playgrounds_Application/Simulators/requirements.txt` I'm not sure if that's a conflict or if there is an easy way to get them to play nice, but PyCharm was having trouble with both declarations being mutually exclusive.

Reply:
Also this can't be removed from here or else `pandas` can't install correctly. Maybe it can be removed from the other requirements and this version bumped to 5.3.1